Zend/zend_compile.c

```diff
@@ -2000,14 +2000,19 @@ ZEND_API int zend_is_smart_branch(const zend_op *opline) /* {{{ */
 static inline uint32_t zend_emit_cond_jump(zend_uchar opcode, znode *cond, uint32_t opnum_target) /* {{{ */
 {
 	uint32_t opnum = get_next_op_number();
-	zend_op *opline;
+	zend_op *opline = CG(active_op_array)->opcodes + opnum -1;
```

Comment:
This will likely cause ubsan warnings on clang. I would suggest only computing opline after the `opnum` check:

```
if (cond->op_type == IS_TMP_VAR && opnum > 0) {
    zend_op *opline = CG(active_op_array)->opcodes + opnum - 1;
    if (...) { ... }
}
```